application-services/custom/camera-management/README.md

```diff
@@ -158,7 +158,17 @@ make run-edge-video-analytics
    export WRITABLE_INSECURESECRETS_CAMERACREDENTIALS_SECRETS_PASSWORD="<password>"
    ```  
 
-#### 3.2 Build and run
+#### 3.2 Configure Default Pipeline
```

Comment:
Add something explaining that by default all cameras on startup and all new cameras will run a default pipeline. then say that the default pipeline can be configured to something else, or disabled by setting the values to empty.